sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/SynchronousMetricStorage.java

```diff
@@ -55,23 +68,20 @@
                 metricDescriptor,
                 () -> sampler.createReservoir(view.getAggregation()));
     return new SynchronousMetricStorage<>(
-        metricDescriptor,
-        aggregator,
-        new InstrumentProcessor<>(aggregator, startEpochNanos),
-        view.getAttributesProcessor());
+        metricDescriptor, aggregator, view.getAttributesProcessor());
   }
 
   SynchronousMetricStorage(
       MetricDescriptor metricDescriptor,
       Aggregator<T> aggregator,
-      InstrumentProcessor<T> instrumentProcessor,
       AttributesProcessor attributesProcessor) {
     this.metricDescriptor = metricDescriptor;
     aggregatorLabels = new ConcurrentHashMap<>();
     collectLock = new ReentrantLock();
```

Comment:
this could be moved up to the declaration, yes?

Reply:
(and the aggregatorLabels as well)

Reply:
Done.